GitHubReactions.ps1

```diff
@@ -59,56 +62,90 @@ filter Get-GitHubReaction
         GitHub.Reaction
 
     .EXAMPLE
-        Get-GitHubReaction -OwnerName Microsoft -RepositoryName PowerShellForGitHub -IssueNumber 157
+        Get-GitHubReaction -OwnerName Microsoft -RepositoryName PowerShellForGitHub -Issue 157
 
-        Gets the reactions for issue 157 from the Microsoft\PowerShellForGitHub project.
+        Gets the reactions for issue 157 from the Microsoft\PowerShellForGitHub
+        project.
 
     .EXAMPLE
-        Get-GitHubReaction -OwnerName Microsoft -RepositoryName PowerShellForGitHub -IssueNumber 157 -ReactionType eyes
+        Get-GitHubReaction -OwnerName Microsoft -RepositoryName PowerShellForGitHub -Issue 157 `
```

Comment:
I really detest the backtick for line splitting commands (even if it's just in sample code in an example).  I'm ok with those example lines specifically going longer...if they have pipe commands, just force the wrap at the pipe and indent the successive lines.